### ISSUE TITLE
A11Y: Mobile view: Close button is not keyboard operable

### ISSUE THREAD

**Issue by 73nici** [shopware/platform#7053]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

Button in the mobile view cannot be triggered using the keyboard. People with motor impairments use a keyboard on mobile devices.

![Image](https://github.com/user-attachments/assets/c2b703ef-647d-4763-ae23-b220f0ae5333)

*_Expected behaviour:_*

Close button in the mobile view can be triggered using the keyboard.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag
- Occurs consistently if you load the page directly in the mobile viewport. If you first load the page in the desktop viewport and then switch to mobile, the button can also be operated using the keyboard.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40819**



[created from NEXT-40819, comment 541471]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40819, comment 541481]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40819, comment 541578]

**Comment by shopwareBot** [comment]
[public] Thank you so much for your detailed report and supporting us to make the internet more accessible! This is already covered by the following issue: [https://github.com/shopware/shopware/issues/6997](https://github.com/shopware/shopware/issues/6997)



[created from NEXT-40819, comment 542347]
